pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "pendulum"
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
FYI: Please don't change the version in the PR like this, a sepearte release PR will make this change

Reply:
sure!